src/Router.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import ProductDetail from './pages/ProductDetail/ProductDetail';
+import ProductListCard from './pages/ProductList/Product/ProductListCard';
+import Footer from './pages/Footer/Footer';
 
 const Router = () => {
   return (
     <BrowserRouter>
       <Routes>
         <Route path="/productdetail" element={<ProductDetail />} />
+        <Route path="/productListCard" element={<ProductListCard />} />
+        <Route path="/footer" element={<Footer />} />
       </Routes>
     </BrowserRouter>
```

Comment:
Footer Component는 
/footer 페이지에 진입했을 때가 아니라 전체 페이지에서 보여줘야 할 것 같아요! 

이렇게 바꾸는건 어떨까요?
```suggestion
    <BrowserRouter>
      <Routes>
        <Route path="/productdetail" element={<ProductDetail />} />
        <Route path="/productListCard" element={<ProductListCard />} />
      </Routes>
      <Footer />
    </BrowserRouter>
```